notes/src/components/ListView.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import NoteCard from './NoteCard'
+import './ListView.css';
+
+const ListView = (state) => (
+  <div className="listContainer">
+    <h2 className="listHeader">Your Notes:</h2>
+    <div className="listBody">
+        {state.noteData.notes.map((note) =>
+          <NoteCard key={note.id} {...note} />
+        )}
+    </div>
+  </div>
+);
+
+const mapStateToProps = (state) => ({
```

Comment:
Another opportunity to use destructuring here! 😆

```
const ListView = ({ notes }) => (
  ...
  {notes.map(note => <NoteCard key={note.id} {...note} />)}
  ...
const mapStateToProps = ({ notes }) => ({ notes })
```

Reply:
Nice! Can't believe I missed that. I was just thinking, "Get in there state! Work!".